### DISCUSSION TITLE
git-lfs and ko interaction

### DISCUSSION THREAD

**Discussion by sam-hoffman** [git-lfs/git-lfs#4748] [Q&A]
Files tracked with git-lfs do not appear to be included as part of the ko build. [This stackoverflow post](https://stackoverflow.com/questions/62823944/how-to-download-a-specific-git-lfs-file-into-a-docker-container) suggests a solution for a similar problem with Docker - is there a recommended solution for ko?

**Comment by bk2204** [comment]
Hey,

I'm not familiar with `ko`.  I assume you're talking about [this project](https://github.com/google/ko).  The recommended solution mentioned for that Stack Overflow post is to not clone with `--no-checkout` if you want your Git LFS files to be checked out, because `git lfs pull` needs to use the working tree (since it implicitly means `git lfs fetch` and then `git lfs checkout`).

If that's the problem that you're seeing, then I'd propose the same solution.  If there's something else and you're seeing an error message, go ahead and post it here and we'll try to look at it.
